projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -3,7 +3,7 @@ import { waitForPage } from '../../../helpers/checkout-flow';
 import { navigation } from '../../../helpers/navigation';
 import { cdsHelper } from '../../../helpers/vendor/cds/cds';
 import { profileTagHelper } from '../../../helpers/vendor/cds/profile-tag';
-describe('Profile-tag component', () => {
+describe.skip('Profile-tag events', () => {
```

Comment:
we do not need to skip anymore :)! 

Dev-17 became our uber server that has everything including cds, and it is set up for it already